src/VisualStudio/Core/Def/Implementation/LanguageClient/LiveShareLanguageServerClient.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.LanguageServer;
+using Microsoft.CodeAnalysis.Remote;
+using Microsoft.VisualStudio.LanguageServer.Client;
+using Microsoft.VisualStudio.Threading;
+using Microsoft.VisualStudio.Utilities;
+using Nerdbank.Streams;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.LanguageService
+{
+    [ContentType("C#_LSP")]
+    [ContentType("VB_LSP")]
```

Comment:
We don't need a content type for FSharp/TypeScript?

Reply:
FSharp was never supported through us for liveshare.

Typescript was, but they are working on their own language server.  So I'm going to leave their roslyn integration as-is (maybe move it to their external access).